R/visr.R

```diff
@@ -316,6 +317,21 @@ visr.survfit <- function(
     ggplot2::theme(legend.key = ggplot2::element_blank()) +
     NULL
   
+  # Save applied function so that we don't have to guess later on
```

Comment:
This is really cool and we should highlight this in the documentation as well. Survminer is taking the lazy route:

  if(is.null(fun)) draw_lines <- TRUE
  else if(fun %in% c("cumhaz", "cloglog")){
    warning("Adding survival median lines is not allowed when fun is: ", fun)
    draw_lines <- FALSE
  }
  else if(fun == "pct") med_y <- 50